.gitlab-ci.yml

```
@@ -175,12 +173,14 @@ test-observability-pipelines-worker:
   tags: ["runner:docker"]
   stage: test
   dependencies: ["generate-scripts"]
+  variables:
+    DD_OPW: 'true'
   parallel:
     matrix:
       - IMAGE: [amazonlinux:2, amazonlinux:2023, debian:10, debian:11, rockylinux:9.0, ubuntu:20.04, ubuntu:22.04]
-        INSTALL_CLASSIC_AGENT: ['', --opw-install-classic-agent install_script_agent7.sh]
+        DD_OPW_INSTALL_CLASSIC_AGENT: ['', install_script_agent7.sh]
   script:
-    - ./test/dockertest.sh --image "registry.ddbuild.io/images/mirror/${IMAGE}" --script install_script_op_worker1.sh --observability-pipelines-worker "true" ${INSTALL_CLASSIC_AGENT}
+    - ./test/dockertest.sh --image "registry.ddbuild.io/images/mirror/${IMAGE}" --script install_script_op_worker1.sh ${INSTALL_CLASSIC_AGENT}
```

```suggestion
    - ./test/dockertest.sh --image "registry.ddbuild.io/images/mirror/${IMAGE}" --script install_script_op_worker1.sh
```

If successfuly figured out how all these variables are used I think we can remove this now, right?

---

Indeed, I just amended the to do so!